addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -283,6 +283,7 @@ def _check_unique_vendor_number(self):
                 ('commercial_partner_id', '=', rec.commercial_partner_id.id),
                 # allow to have to equal if they are cancelled
                 ('state', '!=', 'cancel'),
+                ('l10n_latam_document_type_id', '=', rec.l10n_latam_document_type_id.id),
```

Comment:
@luistorresm I wonder if in some localizations the sequence is not shared among multiple document types.  (which would still work, but not entirely check the right things).  Maybe we can just _prepare this domain and inherit it in l10n_pe? @pepetreshere

Reply:
@jco-odoo @luistorresm, I've just seen this was tagged to me too. I find it nice in the way you already solved this by adding the method _get_domain_unique_vendor_number. So there is nothing else needed than what you already did, and from my side you can close the comment.

In Ecuador, in the future (it is not part of the current MVP), it is probable that we will use same sequence for different document types (for instance a purchace liquidation which is document type 03, and a purchase liquidation for reimbursement wich is document type 41).